robottelo/host_helpers/contenthost_mixins.py

```diff
@@ -40,7 +40,27 @@ def OSCAP(self):
             'cbrhel': constants.OSCAP_PROFILE[f'cbrhel{self._v_major}'],
         }
 
-    # pull from settings.server.repos???
+    @cached_property
+    def dogfood_repofile(self):
+        from robottelo.config import settings
+
+        if self.__class__.__name__ == 'ContentHost':
+            product = 'tools'
+        else:
+            product = self.__class__.__name__.lower()
+        # this may be also available from self.satellite.version
+        split_ver = str(settings.server.version.release).split('.')
```

Comment:
Seems worth trying to use `self.satellite.version` here instead of reading settings.